packages/extension-chakra-storefront/src/components/product-tile/index.jsx

```diff
@@ -35,13 +35,16 @@ import PromoCallout from '../../components/product-tile/promo-callout'
 import {useIntl} from 'react-intl'
 
 // Other
-import {PRODUCT_TILE_IMAGE_VIEW_TYPE, PRODUCT_TILE_SELECTABLE_ATTRIBUTE_ID} from '../../constants'
 import {productUrlBuilder, rebuildPathWithParams} from '../../utils/url'
 import {getPriceData} from '../../utils/product-utils'
 import {useCurrency} from '../../hooks'
 import {filterImageGroups, getDecoratedVariationAttributes} from '../../utils/product-utils'
 import {PRODUCT_BADGE_DETAILS} from '../../constants'
 
+// Constants for product tile page
+export const PRODUCT_TILE_IMAGE_VIEW_TYPE = 'large'
+export const PRODUCT_TILE_SELECTABLE_ATTRIBUTE_ID = 'color'
```

Comment:
Looked at how these const are used, they are used as default value for ProductTile component when the props are undefined. IMO, these values are not likely to change and do not need to be in the config.